unit_tests_metadata.yaml

```diff
@@ -0,0 +1,18 @@
+# this should have the same structure as the expected yaml configs, ex conformance.yaml
```

Comment:
Oh so this gets like summed with whatever we actually extract? That's not "metadata", this is like a patch to apply.

Reply:
Yeah, I changed the name from metadata to patch